pkgs/servers/scylladb/seastar-configure-script-paths.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/seastar/configure.py b/seastar/configure.py
+index 62d9c204..f6520635 100755
+--- a/seastar/configure.py
++++ b/seastar/configure.py
+@@ -924,7 +924,7 @@ with open(buildfile, 'w') as f:
+             command = ragel -G2 -o $out $in && sed -i -e '1h;2,$$H;$$!d;g' -re 's/static const char _nfa[^;]*;//g' $out
+             description = RAGEL $out
+         rule gen
+-            command = /bin/echo -e $text > $out
++            command = echo -e $text > $out
```

Comment:
I can't find /bin/echo in the upstream source, is this patch really needed?

Reply:
Unfortunately, yep, it's well hidden because of git submodules: https://github.com/scylladb/scylla-seastar/blob/f541231a3011420e26fb33e71355415334dcb861/configure.py#L927